test/stools_tests.jl

```diff
@@ -56,18 +56,32 @@ d2 = Dict("x" => 3, "y" => 4)
 @testset "tag! ($(keytype(d)))" for d in (d1, d2)
     @testset "no patch ($(dirty ? "dirty" : "clean"))" for dirty in (true, false)
         path = dirty ? dpath : cpath
-        _test_tag!(d, path, false, nothing) # variable unset
-        _test_tag!(d, path, false, "") # variable set but empty
+        _test_tag!(d, path, false,  nothing) # variable unset
+        _test_tag!(d, path, false,  "") # variable set but empty
         _test_tag!(d, path, false, "false") # variable parses as false
-        _test_tag!(d, path, false, "0") # variable parses as false
-        _test_tag!(d, path, false, "rubbish") # variable not a Bool
+        _test_tag!(d, path, false,  "0") # variable parses as false
+        _test_tag!(d, path, false,  "rubbish") # variable not a Bool
     end
     @testset "patch" begin
         _test_tag!(d, dpath, true, "true") # variable parses as true
-        _test_tag!(d, dpath, true, "1") # variable parses as true
+        _test_tag!(d, dpath, true,  "1") # variable parses as true
+    end
+    @testset "message_name" begin
+        path = mktempdir(cleanup=true) # delete path on process exit
+        repo = LibGit2.init(path)
+        LibGit2.commit(repo, "tmp repo commit")
+        message_commit_test = LibGit2.GitCommit(repo, "HEAD")
+        message_name = LibGit2.message(message_commit_test)
+        if (message_name !== nothing) && (message_name != "")
+            print(message_name)
+       end
```

Comment:
This test block is wrong unfortunately. **You need to test the `tag!` function. This is what DrWatson provides. We don't care about testing Lib2Git.**

Please call the `tag!` function, with the keyword `commit_message` to true. Then manually examine the tagged dictionary, and manually write the test 
```
@test d["gitmessage"] == "the mesage you expecet"
```
Not calling the `@test` macro means you are not testing anything. Printing is not testing.

Reply:
Sorry, this might be a dumb question, but will it also work if instead I add the `@test d["gitmessage"] == "the mesage you expect"` in the `_test_tag!` function?  So something like this? I'm terribly sorry for the beginner mistakes..

<img width="442" alt="image" src="https://github.com/JuliaDynamics/DrWatson.jl/assets/142205831/c8a4efc3-642c-45e8-a919-2503e87e4e41">

Reply:
Don't alter any of the previous code. It is simply not worth it. Why not just add this new test code block, that just has 5 lines of code, and does the test? Simpler = better.

(Sure, there could be ways to alter the previous code, but why do it though? just make a new `@testset` codeblock with 4 lines of code that call the `tag!` function and read the message and call an `@test` on the read message.)

Reply:
Somehow it didn't work for me.............